jwst/pathloss/pathloss.py

```diff
@@ -126,11 +148,15 @@ def calculate_pathloss_vector(pathloss_refdata,
     # pointsource.data is 3-d, so we have to extract a wavelength vector
     # at the specified location.  We do this using bilinear interpolation
     else:
-        crpix3 = pathloss_wcs.crpix3
-        crval3 = pathloss_wcs.crval3
-        cdelt3 = pathloss_wcs.cdelt3
-        for i in np.arange(wavesize):
-            wavelength[i] = crval3 + (float(i + 1) - crpix3) * cdelt3
+
+        # If requested, calculate a wavelength vector from the ref file
```

Comment:
Most minor of nitpicks, but perhaps change wording to indicate the default is on as well? "Unless not requested?" I'm not much of a wordsmith...